pkg/manifests/config.go

```diff
@@ -305,6 +307,10 @@ type Audit struct {
 	Profile auditv1.Level `json:"profile"`
 }
 
+type DedicatedServiceMonitors struct {
+	Enabled *bool `json:"enabled"`
```

Comment:
Agreed, I actually specifically implemented this as a pointer since this seemed to be the convention in the config. But `EtcdConfig` was a poor example to go by :sweat_smile: